### PR TITLE
use shallow clone and pull

### PR DESCRIPTION
**Pull request by heitorPB**

### CODE REVIEW

File: kipi/kicad_getlibs.py [hunk 1]
```diff
@@ -366,15 +366,15 @@ def git_clone_or_update (repo_url, target_path, target_name, git_output):
         else:
             if not args.quiet:
                 print ("Updating repo", repo_name)
-            pr = Popen(["git", "pull"], cwd=os.path.join(target_path, target_name), stdout=git_output)
+            pr = Popen(["git", "pull" "--depth=1",], cwd=os.path.join(target_path, target_name), stdout=git_output)
```
Comment (bobc) [inline kipi/kicad_getlibs.py]: Is comma in right place?

**Comment by bobc** [comment]
I think this is a good idea. There seems to be a small typo though.

**Comment by heitorPB** [comment]
Yes, there was a typo, sorry.

**Comment by bobc** [comment]
Thanks!

**Pull request merged**